### PR TITLE
:sparkles: Add feedback form

### LINKED ISSUE #5: [FEATURE] Feedback form
A feedback form containing questions regarding
- Questions
- Ease of access
- Interface
- Enjoyment
- Would play again?
- Would recommend playing?
- Organizers
- Maintainers
- Overall

etc.

### PR DESCRIPTION
**Pull request by jcombs0929**
FIX #5

### CODE REVIEW

File: feedback.html [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+                                    <option value="">Select Rating</option>
+                                    <option>Excellent</option>
+                                    <option>Average</option>
+                                    <option>Poor</option>
+                                    <option>N/A</option>
+                                </select>
+                                <div class="invalid-feedback">Please choose a rating.</div>
+                            </div>
+                            <div class="col-md-6">
+                                <label for="interface">Interface</label>
```
Comment (saglaniharsh) [inline feedback.html]: Change to: How was the User Interface?
Options: 
- Easy to understand
- Just Right
- Difficult to navigate through.

File: feedback.html [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+                                    <input type="radio" id="recommend-no" name="recommend" class="toggle">
+                                    <label for="recommend-no" class="btn btn-outline-primary toggle-no">No</label>
+                                </div>
+                                <div class="invalid-feedback">Please select Yes/No</div>
+                            </div>
+                        </div>
+
+                        <div class="row mb-4">
+                            <div class="col-md-12">
+                                <label for="organizers">Who were the organizers?</label>
```
Comment (saglaniharsh) [inline feedback.html]: The question wasn't who were the organizers xD
It is supposed to be: How supportive were the organizers?
Options:
Very supportive
Just right
Not at all

File: feedback.html [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+                                <input type="text" class="form-control" id="organizers" placeholder="Add Organizers">
+                            </div>
+                            <div class="invalid-feedback">Invalid</div>
+                        </div>
+
+                        <div class="row mb-4">
+                            <div class="col-md-12">
+                                <label for="maintainers">Maintainers?</label>
```
Comment (saglaniharsh) [inline feedback.html]: Same as organizers

Comment (ashawe) [inline feedback.html:133]: Same as organizers

File: feedback.html [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+                                    <option>Poor</option>
+                                    <option>N/A</option>
+                                </select>
+                                <div class="invalid-feedback">Please choose a rating</div>
+                            </div>
+                    
+                        </div>
+                        <div class="row mb-3">
+                            <div class="col-md-12 mb-3">
+                                <label for="additionComments">Any additional comments or questions?</label>
```
Comment (saglaniharsh) [inline feedback.html]: also ask about any difficulties faced

**Review by saglaniharsh — CHANGES_REQUESTED** [review]
Add these things:
How would you rate the questions?
- easy to solve
- just right
- difficult to solve
- difficult to decipher the question

How would you rate the clues / hints?
- easy to understand
- made the questions too easy
- just right
- did not help

File: feedback.html (hunk 4 above)
Comment (saglaniharsh) [reply feedback.html]: in the same question

**Comment by jcombs0929** [comment]
@ashawe Can you clarify what you mean here?

**Comment by jcombs0929** [comment]
@ashawe Did you want a separate textarea for difficulties faced, did you want it added to Additional Comments question?

**Comment by saglaniharsh** [comment]
> 
> @ashawe Did you want a separate textarea for difficulties faced, did you want it added to Additional Comments question?

nope in the same

**Comment by jcombs0929** [comment]
@ashawe Sorry, can you clarify what you mean here?
> > +                            <div class="col-md-12">
> +                                <label for="overallExperience">How would you rate your overall experience?</label>
> +                                <select class="custom-select d-block w-100" id="overallExperience">
> +                                    <option value="">Select Overall Experience</option>
> +                                    <option>Excellent</option>
> +                                    <option>Average</option>
> +                                    <option>Poor</option>
> +                                    <option>N/A</option>
> +                                </select>
> +                                <div class="invalid-feedback">Please choose a rating</div>
> +                            </div>
> +                    
> +                        </div>
> +                        <div class="row mb-3">
> +                            <div class="col-md-12 mb-3">
> +                                <label for="additionComments">Any additional comments or questions?</label>
> in the same question
>

**Comment by saglaniharsh** [comment]
@jcombs0929 change the question from 
Any additional comments or questions?
to
Did you face any difficulties? or have additional comments or questions?

**Comment by jcombs0929** [comment]
@ashawe I have committed my changes based on your feedback. Let me know if you have any other edits.

**Review by saglaniharsh — APPROVED** [review]

**Pull request merged**